test/ModuleInterface/layout_pre_specialization.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -enable-experimental-feature LayoutPrespecialization -typecheck -module-name LayoutPrespec -emit-module-interface-path %t/LayoutPrespec.swiftinterface %s
```

Comment:
For module interface tests in general I recommend using the `%target-swift-emit-module-interface` and `%target-swift-typecheck-module-from-interface` lit substitutions as they set up a few options that should be used in conjunction with producing an interface like `-enable-library-evolution` and `-swift 5`.